// *****************************************************************************
//
// This file is part of a MASA library or program.
// Refer to the included end-user license agreement for restrictions.
//
// Copyright (c) 2010 MASA Group
//
// *****************************************************************************

package MsgsClientToAar;

message MsgPlotRequest {
   required uint32 identifier = 1;
   required string request    = 2;
}

message MsgClientToAar {
    message Content {
        optional MsgPlotRequest plot_request = 1;
    }
    optional int32   context = 1 [default = 0];
    required Content message = 2;
}
